backEnd/src/main/java/handlers/GetGroupHandler.java

```diff
@@ -41,6 +38,7 @@ public GetGroupHandler(final DbAccessManager dbAccessManager, final Metrics metr
    */
   public ResultStatus handle(final String activeUser, final String groupId,
       final Integer batchNumber) {
+    //todo remove batch number param

```

Comment:
Actually this should stay since on the frontend it refreshes the page by calling getGroup. Unless we want to make a separate method for refreshing.

Reply:
I'll leave it until the next pr when this flow gets implemented.